apps/sports-helsinki/src/utils/routerUtils.ts

```diff
@@ -82,17 +82,18 @@ export const removeQueryParamsFromRouter = (
   const queryObject = { ...router.query };
 
   if (removeList.length > 0) {
-    removeList.forEach((param) => delete router.query[param]);
+    removeList.forEach((param) => delete queryObject[param]);
```

Comment:
the method name is `removeQueryParamsFromRouter` but now only cloned `queryObject` is changed but not the original query object? Was that on purpose?

Reply:
I actually don't know what has been the original purpose of this, but earlier it did not work and now it does 😄 . I think the old command was done for react-router and not for next-router and therefore it worked differently. The thing was that nothing was actually done to queryObject that was used in the next router push-command.